### PR TITLE
Replace swsssdk.SonicV2Connector with swsscommon.SonicV2Connector (SWIG wrapper of C++ implementation) in production code

### PR DESCRIPTION
**Pull request by qiluo-msft**
This is the following step for merging sonic-py-swsssdk into sonic-swss-common https://github.com/Azure/sonic-swss-common/pull/387

**Please merge after sonic-swss-common https://github.com/Azure/sonic-swss-common/pull/387 merged**
**Please merge after sonic-swss-common https://github.com/Azure/sonic-swss-common/pull/401 merged**

Note: Test code could still using old python implementation

### CODE REVIEW

**Comment by qiluo-msft** [comment]
retest this please

**Comment by qiluo-msft** [comment]
retest this please

**Comment by qiluo-msft** [comment]
retest this please

**Comment by qiluo-msft** [comment]
retest this please

File: tests/mock_tables/dbconnector.py [hunk 1]
```diff
@@ -143,3 +146,10 @@ def keys(self, pattern='*'):
 mockredis.MockRedis.config_set = config_set
 redis.StrictRedis = SwssSyncClient
 SonicV2Connector.connect = connect_SonicV2Connector
+swsssdk.SonicV2Connector = SonicV2Connector
```
Comment (SuvarnaMeenakshi) [inline tests/mock_tables/dbconnector.py:149]: Why is this required? Isn't SonicV2Connector already swsssdk.SonicV2Connector?

Comment (qiluo-msft) [reply tests/mock_tables/dbconnector.py]: Removed

---
In reply to: [518283029](https://github.com/Azure/sonic-snmpagent/pull/162#discussion_r518283029) [](ancestors = 518283029)

**Review by SuvarnaMeenakshi — APPROVED** [review]
